src/main/java/seedu/address/model/project/Deadline.java

```diff
@@ -0,0 +1,57 @@
+package seedu.address.model.project;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a Project's deadline.
+ */
+public class Deadline {
+
+    /**
+     * Represents an empty Project deadline.
+     */
+    private static class EmptyDeadline extends Deadline {
+        private static final Deadline EMPTY_DEADLINE = new EmptyDeadline();
```

Comment:
Should there be some method to access this variable from outside `Deadline` and some way to validate that it is indeed an `EmptyDeadline`? Consider a `static` method or variable in Deadline. Implementation can be moved to v1.2b when the static variables are used in the command parser.

Reply:
May self-merge PR after reviewing this comment and fixing CI checks.